pkg/storage/sqlite/schema/defaults.sql

```diff
@@ -0,0 +1,56 @@
+-- Inserting into the quality_definition table
+INSERT INTO quality_definition (quality_id, name, preferred_size, min_size, max_size, media_type) 
+VALUES 
+(1, 'HDTV-720p', 1999, 17.1, 2000, 'movie'),
+(2, 'WEBDL-720p', 1999, 12.5, 2000, 'movie'),
+(3, 'WEBRip-720p', 1999, 12.5, 2000, 'movie'),
+(4, 'Bluray-720p', 1999, 25.7, 2000, 'movie'),
+(5, 'HDTV-1080p', 1999, 33.8, 2000, 'movie'),
+(6, 'WEBDL-1080p', 1999, 12.5, 2000, 'movie'),
+(7, 'WEBRip-1080p', 1999, 12.5, 2000, 'movie'),
+(8, 'Bluray-1080p', 1999, 50.8, 2000, 'movie'),
+(9, 'Remux-1080p', 1999, 102, 2000, 'movie'),
+(10, 'HDTV-2160p', 1999, 85, 2000, 'movie'),
+(11, 'WEBDL-2160p', 1999, 34.5, 2000, 'movie'),
+(12, 'WEBRip-2160p', 1999, 34.5, 2000, 'movie'),
+(13, 'Bluray-2160p', 1999, 102, 2000, 'movie'),
+(14, 'Remux-2160p', 1999, 187.4, 2000, 'movie');
+
+INSERT INTO "quality_profile" ("name", "cutoff", "upgrade_allowed") 
+VALUES 
+('Standard Definition', 2, TRUE),
+('High Definition', 8, TRUE),
+('Ultra High Definition', 13, FALSE);
+
+INSERT INTO "quality_item" ("quality_id", "name", "allowed", "parent_id") 
```

Comment:
Do we need a `quality_item`? `profile_quality_item` could link a profile to the definition directly. I'm probably just unclear what allowed and parent_id are for though. What type of item would `parent_id` point to?

Reply:
yeah I don't think we do, I ended up removing this as part of the schema and directly referencing the quality definition in the profiles instead